regolith/schemas.py

```diff
@@ -356,6 +356,49 @@
         "year": 2020,
         "zip": "10027",
     },
+    {
```

Comment:
Please check if there is a meeting schema in my sm_grp_mtgs PR that we are working to merge.  If there is, we should probably use that, if not you can start a meetings schema here and we can get this merged, then use this schema in that branch with a rebase.

Reply:
@sbillinge we couldn't find the PR, could you take a look?

Reply:
#411